pkgs/servers/irc/robustirc-bridge/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Could you please switch to `buildGoModule`? Upstream seems to support it - https://github.com/robustirc/bridge/blob/v1.8/go.mod .

Reply:
I did it and it works. Thanks for the hint.